spring-cloud-stream/src/main/java/org/springframework/cloud/stream/aggregate/AggregateApplicationBuilder.java

```diff
@@ -386,15 +424,7 @@ void embed() {
 							e);
 				}
 			}
-			// Register metrics if JMX enabled and exporter avalable
-			if (BeanFactoryUtils.beansOfTypeIncludingAncestors(AggregateApplicationBuilder.this.parentContext,
```

Comment:
Given all the metrics are now exposed via `/<namespace>/metrics` endpoint, I believe this explicit registration of metrics can be removed.